examples/jetstream/push-consumer/java/Main.java

```diff
@@ -0,0 +1,198 @@
+package example;
+
+import io.nats.client.*;
+import io.nats.client.api.*;
+import io.nats.client.impl.NatsJetStreamMetaData;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.concurrent.CountDownLatch;
+
+public class Main {
+    public static void main(String[] args) {
+        String natsURL = System.getenv("NATS_URL");
+        if (natsURL == null) {
+            natsURL = "nats://127.0.0.1:4222";
+        }
+
+        // Initialize a connection to the server. The connection is AutoCloseable
+        // on exit.
+        try (Connection nc = Nats.connect(natsURL)) {
+
+            // Access `JetStream` and `JetStreamManagement` which provide methods to create
+            // streams and consumers as well as convenience methods for publishing
+            // to streams and consuming messages from the streams.
+            JetStream js = nc.jetStream();
+            JetStreamManagement jsm = nc.jetStreamManagement();
+
+            // Declare a simple [limits-based stream][1] and populate the stream
+            // with a few messages.
+            // [1]: /examples/jetstream/limits-stream/java/
```

Comment:
I think this can just be a simple comment about creating the stream. I say this not just because we would have to go fix other examples, but when we make an example, I think want to discuss/introduce concepts that absolutely are required for the example. The limit based stream concept is a side quest we don't need here.

Reply:
Changed the comment to be just:
```
// Declare a simple stream and populate it with a few messages.
```